.github/workflows/test-cosmwasm-onchain.yml

```
@@ -0,0 +1,338 @@
+      # Checkout wasm-testing repository
+      - name: Checkout wasm-testing
+        uses: actions/checkout@v4
+        with:
+          repository: MANTRA-Chain/wasm-testing
+          path: wasm-testing
+          ref: main
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Set up Rust environment
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)